README.md

```diff
@@ -1,5 +1,8 @@
 # project-sandbox
-a playground for experimentation
+* Project Sandbox will explore mapping the uses of publicly-owned vacant properties owned by the City of Los Angeles.
```

Comment:
"Project Sandbox's goal is to map and explore the uses of publicly-owned vacant properties owned by the City of Los Angeles"?